src/server/pps/server/api_server.go

```diff
@@ -2520,7 +2524,7 @@ func (a *apiServer) InspectPipelineInTransaction(txnCtx *txncontext.TransactionC
 	pipelineInfo := &pps.PipelineInfo{}
 	if err := a.pipelines.ReadWrite(txnCtx.SqlTx).Get(commit, pipelineInfo); err != nil {
 		if col.IsErrNotFound(err) {
-			return nil, errors.Errorf("pipeline %s not found", pipeline)
+			return nil, errors.Errorf("pipeline %s not found with spec commit %q", pipeline, pfsdb.CommitKey(commit))
```

Comment:
I wonder if the `*Key` methods should be `String` methods on the objects, and the `Parse*` functions should live in packages `pfs` & `pps`?